content/blog/how-to-use-dsc-logging.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "How to Use Dsc Logging"
+date: 2019-09-24T16:37:42+02:00
+type: "post"
+draft: false
+---
+
+PowerShell Desired State Configuration (DSC) logs its deployment progress and errors
```

Comment:
I'd specify that's the LCM, because technically that changes with PS7's Invoke-DscResource.